src/i18n/lang/pl-PL.lang.ts

```diff
@@ -0,0 +1,89 @@
+	public override language_change_success = (oldLocale: Nullable<I18NLanguage>, newLocale: I18NLanguage): string =>
+		`Pomyślnie zmieniono język${oldLocale ? ` z \`${oldLocale}` : ''} na \`${newLocale}\``;
+
+	public override embed_help_callout_title = 'Potrzebujesz pomocy?';
+	// TODO: branding?
+	public override embed_remoji_author_name = 'Remoji - Discord emoji manager';
+	public override embed_footer_tagline = 'Remoji - Discord emoji manager - Stworzony przez Shino';
+	public override embed_join_support_server_link = 'Dołącz do serwera';
+	public override embed_vote_callout_link = 'Kliknij tutaj aby głosować na Remoji!';
+
+	// TODO: guild only or maybe not in DMs?
+	public override command_error_guild_only = ':x: Ta komenda może być użyta jedynie na serwerach.';
+	public override command_error_user_missing_permission = (permissions: PermissionString[]): string =>
+		`Aby użyć tej komendy, potrzebujesz tych uprawnień: ${permissions.map((perm) => `\`${perm}\``).join(', ')}`;
+	public override command_error_bot_missing_permission = (permissions: PermissionString[]): string =>
+		`Aby użyć tej komendy, Remoji potrzebuje tych uprawnień: ${permissions.map((perm) => `\`${perm}\``).join(', ')}`;
+	// TODO: unlock? use?
+	public override command_error_vote_locked = (commandName: string, topggURL: string): string =>
+		`:lock: aby odblokować komendę \`/${commandName}\`, [zagłosuj na Remoji na top.gg](${topggURL})!`;
+
+	public override emote_copy_invalid_emote =
+		'Nie znaleziono emoji do skopiowania. Można kopiować tylko **niestandardowe** emoji.';
+	// TODO: emoji? emote?
+	public override emote_copy_invalid_name =
+		'To nie jest poprawna nazwa dla emoji. Nazwa musi mieć od 2 do 32 znaków oraz składać się tylko z liter, cyfr lub znaków podkreślenia';
+	// TODO: ?
+	public override emote_copy_invalid_url = 'Nie udało się pobrać emoji. Czy na pewno nie została usunięta?';
+	public override emote_copy_invalid_domain =
+		'Nie udało się pobrać emoji z powodu niepoprawnego adresu CDN. Prosimy o zgłoszenie tego błędu deweloperom.';
+	public override emote_copy_unknown_download_error = 'Podczas pobierania emoji wystąpił nieznany błąd.';
+	public override emote_copy_unknown_upload_error = 'Podczas przesyłania emoji na serwer wystąpił nieznany błąd.';
+	public override emote_copy_success = (name: string, newEmoji: string): string =>
+		`:tada:  Skopiowano \`:${name}:\` na Twój serwer! ${newEmoji}`;
+	public override emote_copy_no_emotes = 'Musisz podać przyynajmniej 1 emoji do skopiowania.';
+	public override emote_copy_multiple_success = (failed: number, success: number): string =>
+		`Pomyślnie przesłano ${success} emoji! (Niepomyślne: ${failed})`;
+
+	// TODO: ????
+	// TODO: Uh, you do know that managing emoji is done in discord anyway, and that since 8.45g from march 2019 android has ways to manage emotes as well, right?
```

Comment:
Yes, I'm aware mobile has this capability; however, there were numerous people in the support server who asked me to "add support for mobile," so I added mobile support to the description.